src/passwordPenalty.js

```diff
@@ -12,16 +12,11 @@ export default function penaltyPoints(password = "") {
   if(password === null) return 0;
   if (typeof password !== "string") password = String(password);
 
-  // * * * * * * * * * * * * * * * * * * * * * * * * * * * *
-  // * * * INSERT YOUR CODE HERE * * * * * * * * * * * * * *
-    //
-    let regex = (/([a-z0-9])\1+/gi)
-    let count = 0;
+    // Here this line of code below groups similar characters together
+    const regex = (/([a-z0-9])\1+/gi)
    let matchArr = (password.match(regex) || 0)
      if(matchArr === 0) return 0;
-   for(let cha of matchArr){
-      if(cha.length === 2) count+=1
-      if(cha.length >= 3) count+=2
-   }
-   return count
+    return matchArr.reduce((count, currentValue) => {
+     return currentValue.length === 2 ? count += 1 : currentValue.length >= 3 ? count+= 2 : count
+    }, 0)
```

Comment:
Can you consider to remove this condition to check if greater than or equal to 3 because it repeating itself

Reply:
- kindly make me understand how it's repeating itself. Because that condition caters for  numbers with length 3 and above and if not included it can cause also characters with length of one to be counted